API/Parser/Parser.cs

```diff
@@ -136,213 +136,121 @@ public static class Parser
         {
             // Grand Blue Dreaming - SP02
             new Regex(
-                @"(?<Series>.*)(\b|_|-|\s)(?:sp)\d",
+                $@"{seriesExpr}(\b|-|\s)(?:sp)\d",
                 MatchOptions, RegexTimeout),
-            // [SugoiSugoi]_NEEDLESS_Vol.2_-_Disk_The_Informant_5_[ENG].rar, Yuusha Ga Shinda! - Vol.tbd Chapter 27.001 V2 Infection ①.cbz
```

Comment:
Please make sure this case is covered in the code
[SugoiSugoi]_NEEDLESS_Vol.2_-_Disk_The_Informant_5_[ENG].rar

Reply:
That's covered by the regex at line 147. Please see comment at line 145.